ragavi/utils.py

```diff
@@ -544,8 +559,11 @@ def __handle_uncaught_exceptions(extype, exval, extraceback):
 
 ########################################################################
 ########################### Some useful functions ######################
+
 def __welcome():
     """Welcome to ragavi"""
+    from pyfiglet import Figlet
```

Comment:
Any reason for this import.

Reply:
It's not exactly in use.

Reply:
Then I guess this should be removed because i see it's not even in the requirements.

Reply:
I've commented it out now. Can I merge?

Reply:
Yeah go ahead :+1: